model-engine/model_engine_server/core/celery/celery_autoscaler.py

```diff
@@ -54,6 +54,7 @@ def excluded_namespaces():
 
 autoscaler_broker = os.environ.get("BROKER_NAME", SQS_BROKER)
 aws_profile = os.environ.get("AWS_PROFILE")
+aws_region = os.environ.get("AWS_REGION", "us-west-2")
```

Comment:
we do have `infra_config().default_region`, this gets (pretty much) passed through either through helm installs or through configs baked into the image (probably doesn't apply for your use case). I'm not sure if we have `AWS_REGION` as an available env var. cc @squeakymouse

Reply:
I think it's currently injected by the IRSA setup

Reply:
So this basically https://github.com/aws/amazon-eks-pod-identity-webhook injects the AWS_REGION in EKS cluster once you setup the service account with a role